test/.secrets.baseline

```diff
@@ -0,0 +1,116 @@
+{
+  "version": "1.4.0",
```

Comment:
Do we need this additional file if we've already got a `.secrets.baseline` in the root folder?

Reply:
Good spot - just removed that file.
We only need the one in the root folder.
Thanks